Dockerfile

```diff
@@ -48,7 +54,9 @@ RUN apt-get update \
     && apt-get install -y pandoc netcat-openbsd \
     && rm -rf /var/lib/apt/lists/*
 
-# RUN python -c "from sentence_transformers import SentenceTransformer; model = SentenceTransformer('all-MiniLM-L6-v2')"
+# preload embedding model
+RUN python -c "import os; from chromadb.utils import embedding_functions; sentence_transformer_ef = embedding_functions.SentenceTransformerEmbeddingFunction(model_name=os.environ['RAG_EMBEDDING_MODEL'])"
```

Comment:
We might want to save the models to `/data/cache/embedding/models`!

Reply:
Ok, added an ENV Variable in the Dockerfile to store the embedding models in this folder, also i added the option to change the device type if possible.

Maybe we should automate the re-imbedd process in future with a script executed in the Dockerfile, because as mentioned changing the embedding model needs to re-imbedd importet docs.